jest/setup.js

```diff
@@ -1,15 +1,143 @@
+import fs from 'fs';
+import path from 'path';
 import 'react-native-gesture-handler/jestSetup';
+import _ from 'underscore';
 
 require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests();
 
 // Silence the warning: Animated: `useNativeDriver` is not supported because the native animated module is missing
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
 
-// Set up manual mocks for methods used in the actions so our test does not fail.
-jest.mock('../src/libs/Notification/PushNotification', () => ({
-    // There is no need for a jest.fn() since we don't need to make assertions against it.
-    register: () => {},
-    deregister: () => {},
+jest.mock('react-native-blob-util', () => ({}));
+
+jest.mock('react-native-reanimated', () => {
+    const Reanimated = require('react-native-reanimated/mock');
+
+    // The mock for `call` immediately calls the callback which is incorrect
```

Comment:
Can you please explain why it's incorrect?

Reply:
Ah ha! This is actually copy/pasta [from the docs](https://reactnavigation.org/docs/testing/#mocking-native-modules). So I'd be inclined to either remove this or add a `- thus speaketh the JS Gods` here.

Reply:
Just removed this and linked to the docs. I don't have a satisfying explanation other than "react navigation made me do it". 🧙